tests/cross/src/test/scala/tests/pc/CompletionScalaCliSuite.scala

```diff
@@ -89,7 +89,7 @@ class CompletionScalaCliSuite extends BaseCompletionSuite {
   check(
     "plugin".tag(
       IgnoreScalaVersion(version =>
-        Set("2.12.12", "2.12.16", "3.2.1")(version) ||
+        Set("2.12.12", "2.12.16", "3.2.2", "3.3.0")(version) ||
```

Comment:
Future proofing since better-to-string might released a bit later than 3.3.0 as with 3.2.2